src/main/java/seedu/address/logic/commands/AddTagsCommand.java

```diff
@@ -84,7 +95,34 @@ public CommandResult execute(Model model) throws CommandException {
         model.setPatient(patientToEdit, editedPatient);
         model.updateFilteredPatientList(PREDICATE_SHOW_ALL_PATIENTS);
 
-        return new CommandResult(String.format(MESSAGE_ADD_TAG_SUCCESS, editedPatient.getName()));
+        logger.log(Level.INFO, "Successfully called AddTagsCommand for patient: " + editedPatient.getName());
+        return new CommandResult(commandResultString);
+    }
+
+    /**
+     * Add specified tags to a patient's tag set.
```

Comment:
According to the Java Convention, the JavaDoc should start with "Adds ..." instead of "Add ...".

Reply:
good catch, resolved!